packages/apt/test.sh

```diff
@@ -185,6 +187,28 @@ if [ "${can_upgrade}" = "yes" ]; then
   run_test
 fi
 
+echo "::group::Downgrade"
+
+if [ "${can_downgrade}" = "yes" ]; then
+  pgroonga_current_version=$(dpkg -l | \
```

Comment:
Is this an updated PGroonga version? I think we want to use a downgrade version here.